post_installation_configuration/coreos-layering.adoc

```diff
@@ -53,24 +86,13 @@ RUN rpm-ostree override replace https://example.com/myrepo/haproxy-1.0.16-5.el8.
 
 * *{op-system-base} packages*. You can download {op-system-base-full} packages from the link:https://access.redhat.com/downloads/content/479/ver=/rhel---9/9.1/x86_64/packages[Red Hat Customer Portal], such as chrony, firewalld, and iputils.
 +
-.Example Containerfile to apply the firewalld utility
-[source,yaml]
-----
-FROM quay.io/openshift-release-dev/ocp-release@sha256...
-ADD configure-firewall-playbook.yml .
-RUN rpm-ostree install firewalld ansible && \
-    ansible-playbook configure-firewall-playbook.yml && \
-    rpm -e ansible && \
-    ostree container commit
-----
-+
-.Example Containerfile to apply the libreswan utility
+.Example out-of-cluster Containerfile to apply the libreswan utility
 [source,yaml]
 ----
 include::https://raw.githubusercontent.com/openshift/rhcos-image-layering-examples/master/libreswan/Containerfile[]
 ----
 +
-Because libreswan requires additional RHEL packages, the image must be built on an entitled {op-system-base} host.
+Because libreswan requires additional RHEL packages, the image must be built on an entitled {op-system-base} host. For RHEL entitlements to work, you must copy the `etc-pki-entitlement` secret into the `openshift-machine-api` namespace. 
```

Comment:
Throughout PR: Can any of the instances of "RHEL" be changed to `{op-system-base-full}` and `{op-system-base}`?